script.js

```diff
@@ -0,0 +1,317 @@
+    p.classList.add('text-danger');
+    p.innerText = achievements;
+    cBody.appendChild(createElem('hr', 'inside'));
+    p = cBody.appendChild(createElem('p', 'card-text'));
+    p.innerText = info;
+    const anchor = cBody.appendChild(createElem('a'));
+    anchor.href = link;
+    anchor.innerText = 'Read More';
+  }
+};
+
+const mobile = () => {
+  if (courseOffer.hasChildNodes()) {
+    courseOffer.innerHTML = '';
+  }
+
+  fundTeam.classList.add('inactive');
+  footSection.classList.add('inactive');
+  const con = courseOffer.appendChild(createElem('div', 'container'));
+  const h2 = con.appendChild(createElem('h2', 'text-center'));
+  h2.classList.add('py-2');
+  h2.innerText = 'Our Courses';
+  con.appendChild(createElem('hr', 'red'));
+  const drow = con.appendChild(createElem('div', 'row'));
+  drow.classList.add('gx-3');
+  drow.classList.add('pt-1');
+  drow.classList.add('pb-5');
+
+  mobileAdd(drow, 0, skills.length / 2);
+
+  const btn = courseOffer.appendChild(createElem('button', 'btn'));
+  btn.classList.add('more');
+  btn.classList.add('p-2');
+  btn.innerText = 'Show More';
+  let sp = btn.appendChild(createElem('span'));
+  let thumb = sp.appendChild(createElem('img'));
+  thumb.src = './assets/images/Disabled.png';
+  thumb.alt = 'Arrow';
+  btn.addEventListener('click', () => {
+    mobileAdd(drow, skills.length / 2, skills.length);
+    btn.innerText = 'Show Less';
+    fundTeam.classList.remove('inactive');
+    footSection.classList.remove('inactive');
+    sp = btn.appendChild(createElem('span'));
+    thumb = sp.appendChild(createElem('img'));
+    thumb.src = './assets/images/Disabled.png';
+    thumb.style.transform = 'rotate(180deg)';
+    btn.addEventListener('click', () => {
+      mobile();
+    });
+  });
+};
+
+if (wSize < 768) {
+  mobile();
+} else {
+  desktop();
+}
+
+let check = false;
+window.onresize = () => {
+  wSize = window.innerWidth;
+  if (wSize < 768 && !check) {
+    check = true;
+    mobile();
+  } else if (wSize >= 768 && check) {
+    check = false;
+    desktop();
+  } else {
+    alert('okay');
+  }
+};
+
+// adding programs dynamically
+const parent = document.querySelector('#programs');
+const con = parent.appendChild(createElem('div', 'container'));
+con.classList.add('pb-5');
+con.classList.add('pt-3');
+const h2 = con.appendChild(createElem('h2', 'text-center'));
+h2.innerText = 'Our Daily Schedule';
+con.appendChild(createElem('hr', 'red'));
+const drow = con.appendChild(createElem('div', 'row'));
+drow.classList.add('justify-content-center');
+drow.classList.add('g-2');
+programs.forEach((program) => {
+  const {
+    icon,
+    head,
+    info,
+  } = program;
+  const dcol = drow.appendChild(createElem('div', 'col-lg-2'));
+  dcol.classList.add('col-md-3');
+  dcol.classList.add('col-sm-4');
+  const ins = dcol.appendChild(createElem('div', 'border'));
+  const img = ins.appendChild(createElem('img', 'img-fluid'));
```

Comment:
- [ ] @MICHAELKITH great work implementing the changes requested however, the design still doesn't match the Behance design check the comparison below:


Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/217657115-29f74413-259e-453b-80ad-8f04e1cc25b1.png)  | ![image](https://user-images.githubusercontent.com/61408860/217657225-7abbebef-e931-4e1c-9001-d6cf487e9707.png)

Reply:
Well done on the changes 💯 almost there but remove the unnecessary border as it doesn't fit the rest of the design 🎨  and also make sure to be consistent with the background, it will look good, I promise. 😉
 
![Screenshot 2023-02-08 at 21 22 03](https://user-images.githubusercontent.com/45739963/217700772-c3a5a1fb-22c2-40be-89b4-cc8bbe7b8a9f.png)